pkg/exchange/okex/types.go

```diff
@@ -0,0 +1,28 @@
+package okex
+
+import "github.com/c9s/bbgo/pkg/fixedpoint"
+
+// login args
+type WebsocketLogin struct {
+	Key        string `json:"apiKey"`
+	Passphrase string `json:"passphrase"`
+	Timestamp  string `json:"timestamp"`
+	Sign       string `json:"sign"`
+}
+
+// op args
+type WebsocketSubscription struct {
+	Channel          string           `json:"channel"`
```

Comment:
replace type string to WebSocketChannelType

Reply:
The reason I do not choose string is ... below is all channel:
```
candle3M candle1M
candle1W
candle1D candle2D candle3D candle5D
candle12H candle6H candle4H candle2H candle1H
candle30m candle15m candle5m candle3m candle1m candle1s
candle3Mutc candle1Mutc candle1Wutc candle1Dutc candle2Dutc candle3Dutc candle5Dutc candle12Hutc candle6Hutc
```
and there is one function `convertIntervalToCandle` to handle this, so I don't change it.